graphstorm-processing/tests/conftest.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
How to trigger this test?

Reply:
`pytest` automatically uses this configuration file when we run `pytest tests/`